docs/javascript/v2/features/change-role.mdx

```diff
@@ -33,20 +33,45 @@ console.log("can I end change role - ", permissions.changeRole);
 const { video, audio, screen } = hmsStore.getState(selectIsAllowedToPublish);
 ```
 
-## Change Role API
+## Change Role APIs
+
+There are two way of chaning peer's role using our api:
+1. [`Single Peer Role API`](#change-single-peer-role-api) - It is useful when we need to change role of a single peer.
+2. [`Bulk Peer Role API`](#change-bulk-peer-role-api) - We could change a role of all the peer to another role. It will be helpful to make breakout rooms, waiting rooms, etc.
```

Comment:
all peers of particular roles to a given role. we should also mention that permissions are to be enabled in dashboard.

Reply:
can we also update the screenshot in this page.

Reply:
Which screenshot?